<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELoad_StatusHomeWid</class>
 <widget class="QWidget" name="ELoad_StatusHomeWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>522</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>总状态显示区</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>最高电压</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="volLab">
          <property name="text">
           <string>--- V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>总电流</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curLab">
          <property name="text">
           <string>--- A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>最高频率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hzLab">
          <property name="text">
           <string>--- </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>总功率</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="powLab">
          <property name="text">
           <string>--- KW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>最高温度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temLab">
          <property name="text">
           <string>--- </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>总电能</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eleLab">
          <property name="text">
           <string>--- KWh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>功能选择区</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>模块设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modeBox">
          <item>
           <property name="text">
            <string>常规动态模式</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>大电流模式</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>手动静态模式</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>时长设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="timeCheckBox">
          <property name="text">
           <string>持续</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="timeSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>分钟</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>输入位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="inputLab">
          <property name="text">
           <string>--- </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="inputCheckBox">
          <property name="text">
           <string>全部</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inputStartSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lab_3">
          <property name="text">
           <string>至</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="inputEndSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>测试时长</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLab">
          <property name="text">
           <string>--- </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>功能操作区</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>统一开关</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openBtn">
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="text">
           <string>关</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
